react-app/src/hooks/useDataLoader.ts

```diff
@@ -54,7 +59,15 @@ const useDataLoader = <AFArgs extends any[], AFResponse = unknown, AFError = unk
     }
   };
 
-  return { refreshData, isLoading, data, error };
+  const loadOnce = async (...args: AFArgs) => {
+    if (oneTimeLoad) {
+      return;
+    }
+    setOneTimeLoad(true);
+    return refreshData(...args);
+  };
+
```

Comment:
A little unsure of what this is doing in the grand scheme of things. Would appreciate more info.

Reply:
Handy for something like the auth user context where we only really need to instance data the one time. This way you can just call loadDataOnce and re-render's wont spam additional requests. You can do something similar by wrapping refreshData in some useEffect/useState stuff where this behavior is desired, but this is just a handy shortcut so you don't have to.